Week2/homework/js-exercises/5_Drink.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+let drinkTray = [];
+const drinkTypes = ['cola', 'lemonade', 'water'];
+
+for (let i=0; i < 5; i++) {
+    if (i<3) {
+        drinkTray.push(drinkTypes[i])
+    } else if (i<4) {
+        drinkTray.push(drinkTypes[i-3])
+    } else {
+        drinkTray.push(drinkTypes[i-3])
+    } 
+}
```

Comment:
Aahaan. You can also make this logic a bit easier by replacing the `if-else-if-else` block with a single statement `drinkTray.push(drinkTypes[i % drinkTypes.length]);`. What do you think? 🙂

Reply:
That's smarter, thanks :)